content/sensu-go/5.0/reference/rbac.md

```diff
@@ -1,85 +1,407 @@
+To create and manage cluster roles, [configure sensuctl][26] as the [default `admin` user][20] or [create a cluster role][25] with permissions for `clusterroles`.
+
+### Default roles
+
+Every [Sensu backend][1] includes:
+
+| Role name       | Type          | Description |
+| --------------- | ------------- | ----------- |
+| `cluster-admin` | `ClusterRole` | Full access to all [resource types][4] across namespaces, including access to [cluster-wide resource types][18].
+| `admin`         | `Role`        | Full access to all [resource types][4] within a namespace. |
+| `edit`          | `Role`        | Read and write access to most resources within a namespace with the exception of roles and role bindings.
+| `view`          | `Role`        | Read-only permission to most [resource types][4] within a namespace with the exception of roles and role bindings.  |
+| `system:agent`  | `ClusterRole` | Used internally by Sensu agents. _WARNING: Modification of this cluster role can result in non-functional Sensu agents._ |
+
+### Viewing roles
+
+You can use [sensuctl][2] to see a list of all roles within Sensu:
+
+{{< highlight shell >}}
+sensuctl role list
+{{< /highlight >}}
+
+To see the permissions and scope for a specific role:
+
+{{< highlight shell >}}
+sensuctl role info admin
+{{< /highlight >}}
+
+To view cluster roles, use the `cluster-role` command:
+
+{{< highlight shell >}}
+sensuctl cluster-role list
+{{< /highlight >}}
+
+### Creating a role
+
+You can use [sensuctl][2] to create a role.
+For example, the following command creates an admin role restricted to the production namespace.
+
+{{< highlight shell >}}
+sensuctl role create prod-admin --create --get --update --delete --namespace production --type *
+{{< /highlight >}}
+
+You can also create a role using a JSON role definition.
+
+{{< highlight shell >}}
+{
+  "name": "prod-admin",
+  "namespace": "production",
+  "rules": [
+    {
+      "verbs": ["create", "read", "update", "delete"],
+      "resources": ["*"],
+      "resourceNames": [""]
+    }
+  ]
+}
+{{< /highlight >}}
+
+### Creating a cluster-wide role
+
+You can use [sensuctl][2] to create a cluster role.
+For example, the following command creates a global event reader role that can read only events across all namespaces within Sensu.
+
+{{< highlight shell >}}
+sensuctl cluster-role create global-event-reader --read --type events
+{{< /highlight >}}
+
+You can also create a role using a JSON role definition.
+
+{{< highlight json >}}
+{
+  "name": "global-event-reader",
+  "rules": [
+    {
+      "verbs": ["get"],
+      "resources": ["events"],
+      "resourceNames": [""]
+    }
+  ]
+}
+{{< /highlight >}}
+
+### Managing roles
+
+You can use [sensuctl][2] to view, create, update, and delete roles.
+To use any of these commands with cluster roles, substitute the `cluster-role` command for the `role` command.
+
+To add permissions to a role:
+
+{{< highlight shell >}}
+sensuctl role add-rule [ROLE-NAME] [flags]
```

Comment:
We have a ton of technical debt in sensuctl so we decided to drop the `add-rule` subcommand and other similar commands. Instead, Eric introduced a new command `sensuctl edit` which could be use to achieve the same result with more consistency. So instead of this subcommand, users could do `sensuctl edit roles [ROLE-NAME] [flags]` and edit the rules with their editor.